sql/sql_audit.cc

```diff
@@ -120,9 +120,10 @@ static audit_handler_t audit_handlers[] =
   general_class_handler, connection_class_handler
 };
 
+#if !defined(DBUG_OFF) && !defined(_lint)
```

Comment:
Why _lint too?

Reply:
I copied this ifdef from the definition of the assert macro, which uses the variable.